cloud_governance/cloud_resource_orchestration/aws/ec2/collect_cro_reports.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime, timedelta
```

Comment:
General remark - For all class method:

1. add typecheck
2. For public method (non private/protected) add logger